lessons/topics/02 - Functions and Prototype/README.md

```diff
@@ -0,0 +1,31 @@
+# Functions and prototypes
+
+In this topic we are going to talk about the different kind of functions, and explain little by litle how to structure our code by adding functions and prototypes, making our code more modular and recyclable
+
+## Topics:
+
+* Functions keywords and structures
+* returns
+* Function composition
+* Function arguments
+* Closures
+* Prototype methodology 
+
+
+## Previous Knowledge
+
+* variables Assignment
+* functions
+* What does the prototype keyword mean in javascript
+* fat arrow functions
```

Comment:
How it is possible to know arrow functions without knowing the functions topics, the same for functions. 

I think that is a lot of confusion that you required previous knowledge about a concept that you actually will cover in this lesson.

Example: how you know suppose to know prototype when this lesson is about this topic.

I think points  2, 3, and 4 should be removed. This lesson should have the content to explain these concepts because this topic is about that in the first place.

Reply:
fixed it, to "what is a variable" the idea is that you, have some idea of what a var is, before heading into the content